pkg/kv/kvserver/replica.go

```diff
@@ -2539,8 +2539,11 @@ func (r *Replica) MeasureRaftCPUNanos(start time.Duration) {
 func (r *Replica) RangeUsageInfo() allocator.RangeUsageInfo {
 	loadStats := r.LoadStats()
 	localityInfo := r.loadStats.RequestLocalityInfo()
+	mvccStats := r.GetMVCCStats()
+	garbagePct := 1 - (float64(mvccStats.LiveBytes) / float64(mvccStats.KeyBytes+mvccStats.ValBytes))
```

Comment:
Adding garbage percentage to the range usage info, as it seems low cost to compute and has been useful in troubleshooting hotspots in the past:

https://cockroachlabs.atlassian.net/wiki/spaces/CKB/pages/2664300986/Playbook+Best+Buy+Case+Study+-+CPU+attribution+to+culprit+queries